tests/protocols/test_http.py

```diff
@@ -61,6 +61,22 @@
 
 CONNECTION_CLOSE_REQUEST = b"\r\n".join([b"GET / HTTP/1.1", b"Host: example.org", b"Connection: close", b"", b""])
 
+CONNECTION_CLOSE_POST_REQUEST_WITH_BODY = b"\r\n".join(
+    [
+        b"POST / HTTP/1.1",
+        b"Host: example.org",
+        b"Connection: close",
+        b"Content-Type: application/json",
+        b"Content-Length: 18",
+        b"",
+        b"{'hello': 'world'}",
+    ]
+)
+
+CONNECTION_CLOSE_POST_REQUEST_WITHOUT_BODY = b"\r\n".join(
```

Comment:
You can remove this one. The above case is fine.

Reply:
Remove this case of POST without body. Kept POST with body and connection close header